From 0c7eda1ae1f9827ce5108bddac07b1d0c22c38dd Mon Sep 17 00:00:00 2001
From: Ricardo Ribalda Delgado <ricardo.ribalda@gmail.com>
Date: Tue, 18 Sep 2018 09:16:40 +0200
Subject: [PATCH] examples/cmake: Add missing targets

felica-lite-dump, felica-read-ndef and mifare-ultralightc-diversify were
present on autoconf but not on cmake

Signed-off-by: Ricardo Ribalda Delgado <ricardo.ribalda@gmail.com>
---
 examples/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index 575bafe..c8e1094 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -1,4 +1,6 @@
 set(EXAMPLES-SOURCES
+	felica-lite-dump
+	felica-read-ndef
         mifare-classic-format
         mifare-classic-read-ndef
         mifare-classic-write-ndef
@@ -12,6 +14,7 @@ set(EXAMPLES-SOURCES
         mifare-desfire-ev1-configure-default-key
         mifare-desfire-ev1-configure-random-uid
         mifare-ultralight-info
+        mifare-ultralightc-diversify
         ntag-detect
         ntag-removeauth
         ntag-setauth
